3ds/flite/PKGBUILD

```diff
@@ -0,0 +1,40 @@
+pkgname=3ds-flite
+pkgver=master
+pkgrel=1
+pkgdesc='A small fast portable speech synthesis system'
+arch=('any')
+url='https://github.com/festvox/flite'
+license=('MIT')
+options=(!strip libtool staticlibs)
+depends=()
+makedepends=('3ds-pkg-config' 'devkitpro-pkgbuild-helpers')
+
+source=("https://github.com/festvox/flite/archive/$pkgver.zip")
```

Comment:
Should be http://festvox.org/flite/packed/flite-${pkgver}/flite-${pkgver}-release.tar.bz2

Reply:
WinterMute wouldn't it be better to keep up with the github repository?